src/s2geography/wkt-writer_test.cc

```diff
@@ -12,17 +12,16 @@ TEST(WKTWriter, SignificantDigits) {
   auto geog = reader.read_feature("POINT (0 3.333333333333334)");
 
   WKTWriter writer_default;
-  EXPECT_EQ(writer_default.write_feature(*geog), "POINT (0 3.333333333333334)");
+  EXPECT_EQ(writer_default.write_feature(*geog), "POINT (0 3.3333333333333344)");
 
   WKTWriter writer_6digits(6);
-  EXPECT_EQ(writer_6digits.write_feature(*geog), "POINT (0 3.33333)");
-
+  EXPECT_EQ(writer_6digits.write_feature(*geog), "POINT (0 3.333333)");
```

Comment:
So this is the main difference I see compared to the previous WKTWriter implementation (maybe because of using ryu?), so that the significant digits are interpreted differently. 

(actually, the `significant_digits` name is then not really correct any more, and should we use `precision` here as well like is used in geoarrow-c?)